doc/python-spglib.md

```diff
@@ -2,102 +2,105 @@
 
 # Spglib for Python
 
-## Installation
+## Usage note
 
-The source code is downloaded at
-<https://github.com/spglib/spglib/tags>.
-But minor updates are not included in this package. If you want the
-latest version, you can git-clone the spglib repository
+The python binding will first try to use your system installed version of spglib. This can
+be controlled by setting an appropriate `LD_LIBRARY_PATH` (linux) or `DYLD_LIBRAY_PATH`
+(macos) to the appropriate location of the `libsymspg.so` library file. If no suitable
+version is found, it will default to using the bundled version.
 
-```shell
-% git clone https://github.com/spglib/spglib.git
-```
+## Installation
 
-It is also possible to install spglib for python via the following
-package distribution services or from building using setup.py.
+The main installation method we support are using [pip](#using-pip) and [conda](#using-conda).
```

Comment:
The crossrefs to `using-pip` and `using-conda` do not work.

Reply:
Does standard markdown not work on this? What is the linked url and real reference url?

Reply:
It seems. We need to use [Myst's cross-referencing](https://myst-parser.readthedocs.io/en/latest/syntax/syntax.html#targets-and-cross-referencing): 

Target header
```markdown
(using-pip)=
### Using pip
```

Referencing
```markdown
{ref}`pip <using-pip>` 
```

Reply:
I mean in the link and reference url in the generated documentation. It says you can use plain markdown, but I'm not sure which side is not working

Reply:
Oh. I mean the standard markdown (referencing `using-pip` head by `[](#using-pip)`) does not work in generated documents by sphinx. Myst complains reference targets are not found like
```
WARNING: 'myst' reference target not found: #using-conda
```

Adding target header (starting with #)
```
(#using-pip)=
### Using pip
```
and referencing it by
```
[pip](#using-pip)
```
seem to work both GitHub's view and generated documents by sphinx.

Reply:
Oh, the documentation says we can add an option to auto-generate the targets. That should make it equivalent on github's view or local ides as well.

Reply:
Ok, the links should be fixed now. check the generated docs in action status below